frontend/src/components/ResizeTableWrapper/ResizableHeader.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
can you please remove the eslint disabling of the rule

Reply:
Done